### PR TITLE
Image classification & metrics updates

### LINKED ISSUE #2453: Pearson Correlation needs documentation
Sentence similarity used Pearson Correlation us an optimizing metric, However, we do not have any documentation for the user that explains or deals with Pearson Correlation.

### LINKED ISSUE #34006: Remove Image Classification Only instructions
With the new Model Builder release, there's no need to install a separate GPU extension. Remove it from the doc.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 0495cff9-97a6-68ba-504c-ec7fcbc69a9c
* Version Independent ID: 0c7c0298-37e3-f945-fee5-fc6ccd7a362d
* Content: [How to install GPU support in Model Builder - ML.NET](https://learn.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/install-gpu-model-builder)
* Content Source: [docs/machine-learning/how-to-guides/install-gpu-model-builder.md](https://github.com/dotnet/docs/blob/main/docs/machine-learning/how-to-guides/install-gpu-model-builder.md)
* Product: **dotnet-ml**
* GitHub Login: @luisquintanilla
* Microsoft Alias: **luquinta**

### PR DESCRIPTION
**Pull request by luisquintanilla**
## Summary

Fixes #34006
Fixes dotnet/machinelearning-modelbuilder#2453

[Preview Link](https://review.learn.microsoft.com/dotnet/machine-learning/how-to-guides/install-gpu-model-builder?branch=pr-en-us-34007)

### CODE REVIEW

**Review by beccamc — APPROVED** [review]

**Comment by luisquintanilla** [comment]
@beccamc I added Pearson Correlation to this PR. Can you please review when you get a chance. Thanks.

**Comment by beccamc** [comment]
@luisquintanilla What does a value of -1 indicate?

**Comment by luisquintanilla** [comment]
@gewarren can I please get an approval on this when you get a chance. Thank you.

File: docs/machine-learning/resources/metrics.md [hunk 1]
```diff
@@ -90,3 +90,9 @@ For further details on regression metrics, read the following articles:
 |----------|-----------------------|-----------|
 |**Area Under ROC Curve**|Area under the receiver operator curve measures how well the model separates anomalous and usual data points.|**Values closer to 1 are better**. Only values greater than 0.5 demonstrate effectiveness of the model. Values of 0.5 or below indicate that the model is no better than randomly allocating the inputs to anomalous and usual categories|
 |**Detection Rate At False Positive Count**|Detection rate at false positive count is the ratio of the number of correctly identified anomalies to the total number of anomalies in a test set, indexed by each false positive. That is, there is a value for detection rate at false positive count for each false positive item.|**Values closer to 1 are better**. If there are no false positives, then this value is 1|
+
+## Evaluation metrics for Sentence Similarity
```
Comment (gewarren) [inline docs/machine-learning/resources/metrics.md:94]:
```suggestion
## Evaluation metrics for sentence similarity
```

File: docs/machine-learning/resources/metrics.md [hunk 2]
```diff
@@ -90,3 +90,9 @@ For further details on regression metrics, read the following articles:
...
+
+| Metric   |      Description      |  Look for |
+|----------|-----------------------|-----------|
+| **Pearson Correlation** |  [Pearson correlation](https://en.wikipedia.org/wiki/Pearson_correlation_coefficient) also known as correlation coefficient measures the dependence or relationship between two sets of data. | **Absolute values closer to 1 are most similar**. This metric ranges from -1 to 1. An absolute value of 1 implies both datasets are identical. A value of 0 implies there is no relationship between the two sets of data.  |
```
Comment (gewarren) [inline docs/machine-learning/resources/metrics.md:98]:
```suggestion
| **Pearson Correlation** | [Pearson correlation](https://en.wikipedia.org/wiki/Pearson_correlation_coefficient), also known as correlation coefficient, measures the dependence or relationship between two sets of data. | **Absolute values closer to 1 are most similar**. This metric ranges from -1 to 1. An absolute value of 1 implies that the datasets are identical. A value of 0 implies there is no relationship between the two sets of data.  |
```

**Review by gewarren — APPROVED** [review]
A couple of suggestions to consider.

**Pull request merged**